specs/lib/ForEachSpec.js

```diff
@@ -402,18 +402,18 @@ describe('ForEach', () => {
                     extras: {
                         compressedImage3DTiles: {
                             s3tc: {
-                                uri: 's3tc.ktx'
+                                uri: 's3tc.ktx2'
                             },
                             etc1: {
-                                uri: 'etc1.ktx'
+                                uri: 'etc1.ktx2'
                             }
                         }
                     }
                 }
             ]
         };
         const returnValue = ForEach.compressedImage(gltf.images[0], (compressedImage, type) => {
```

Comment:
I would actually just purge `ForEach.compressedImage` from the pipeline entirely. Originally we added `extras.compressedImage3DTiles` because we didn't want to commit to a proper glTF extension for compressed textures, but it never really got used.

.ktx2 files will be delivered as https://github.com/KhronosGroup/glTF/pull/1751, which I don't think requires any code changes here except in `removeUnusedElements`. See how WebP is handled for comparison.